.github/workflows/jmh-bechmarks.yml

```diff
@@ -76,6 +76,9 @@ jobs:
       SPARK_LOCAL_IP: localhost
     steps:
     - uses: actions/checkout@v2
+      with:
+        repository: ${{ github.event.inputs.repo }}
+        ref: ${{ github.event.inputs.ref }}
```

Comment:
@nastra I run this in my fork, but the [result](https://github.com/zhongyujiang/iceberg/actions/runs/1733298769) shows  it didn't use the ref i chose. And I found job `run-benchmark`  haven't specify repo and ref here.

Reply:
And this [run](https://github.com/zhongyujiang/iceberg/actions/runs/1733299718)  shows benchmarks are working.

Reply:
good catch!